Binary files orig/gdalraster/src/.Makevars.win.swp and patched/gdalraster/src/.Makevars.win.swp differ
diff -Nru orig/gdalraster/src/Makevars.win patched/gdalraster/src/Makevars.win
--- orig/gdalraster/src/Makevars.win	2023-05-26 21:53:59.000000000 +0200
+++ patched/gdalraster/src/Makevars.win	2023-10-10 10:50:55.095616600 +0200
@@ -3,9 +3,10 @@
 
 TARGET = lib$(subst gcc,,$(COMPILED_BY))$(R_ARCH)
 
-LIBSHARPYUV = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libsharpyuv.a),-lsharpyuv),)
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  LIBSHARPYUV = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libsharpyuv.a),-lsharpyuv),)
 
-PKG_LIBS = \
+  PKG_LIBS = \
 	-fopenmp -lgdal -larmadillo -lopenblas -lgfortran -lquadmath -lpq -lpgcommon \
 	-lpgport -lodbc32 -lodbccp32 -lblosc -lkea -lhdf5_cpp -lhdf5 -lpoppler -llcms2 \
 	-lfreetype -lharfbuzz -lfreetype -llz4 -lpcre2-8 -lxml2 -lopenjp2 -lnetcdf \
@@ -16,6 +17,9 @@
 	-lunistring -liconv -lgcrypt -lcrypto -lgpg-error -lws2_32 -ltiff -llzma \
 	-ljpeg -lz -lcfitsio -lzstd -lwebpdecoder -lwebp $(LIBSHARPYUV) -lsbml-static \
 	-lgeotiff -lproj -lsqlite3 -lbz2 -lcrypt32 -lwldap32 -lsecur32
+else
+  PKG_LIBS = $(shell pkg-config --libs gdal)
+endif
 
 	
 all: clean winlibs
